tests/Stubs/QueueJobWithSavedState.php

```diff
@@ -0,0 +1,62 @@
+<?php
+
+namespace AvtoDev\AmqpRabbitLaravelQueue\Tests\Stubs;
+
+use Illuminate\Queue\InteractsWithQueue;
+use Illuminate\Contracts\Events\Dispatcher;
+use AvtoDev\AmqpRabbitLaravelQueue\WithJobStateTrait;
+use AvtoDev\AmqpRabbitLaravelQueue\StoreStateInterface;
+use AvtoDev\AmqpRabbitLaravelQueue\Tests\Sharer\Sharer;
+
+class QueueJobWithSavedState extends SimpleQueueJob implements StoreStateInterface
+{
+    use WithJobStateTrait, InteractsWithQueue;
+
+    /**
+     * Magic name of state name.
+     */
+    public const MAGIC_PROPERTY = 'magic_counter';
```

Comment:
del, change to hardcoded like `-trigered`

Reply:
`changed`